.gitignore

```diff
@@ -1,2 +1,3 @@
 *.pass
 *.env
+**/.DS_Store
```

Comment:
I'm fine with adding this for any potential macOS contributor. I was looking at comments below [this stack overflow answer](https://stackoverflow.com/a/63517021/9307498) and it seems like just `.DS_Store` would be sufficient to exclude everywhere even in subdirectories? Can you test on your side?

By the way, this is probably something you want to add to your machine's global gitignore as well